src/main/java/edu/ucsb/cs156/example/entities/HelpRequest.java

```diff
@@ -0,0 +1,31 @@
+package edu.ucsb.cs156.example.entities;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Entity;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "helprequest")
```

Comment:
Entity nameshould be `"helprequests"` but otherwise lgtm!

Reply:
Fixed changes waiting for reapproval.